test/controllers/badge.controller.test.js

```diff
@@ -745,13 +740,14 @@ describe('Badge Controller', () => {
       context('when role is an invalid number', () => {
         it('returns error', (done) => {
           request()
-            [method](endpoint)
+            [method](`${endpoint}6`)
```

Comment:
This is very confusing and would be difficult to understand in the future. You should assign that number to a constant before using it:

```suggestion
[method](`${endpoint}${INVALID_ROLE}`)
```